catalog/transformer/catalog-transformer-mgmp/src/test/java/org/codice/alliance/catalog/transformer/mgmp/TestMgmpTransformer.java

```diff
@@ -0,0 +1,369 @@
+
+    private static final String METACARD_DESCRIPTION =
+            "A large scale plan of an airfield outlining information related to that specific airfield.";
+
+    private static final String SERVICE = "service";
+
+    private static final String ISR_CATEGORY_1 = "physicalMeasurement";
+
+    private static final String ISR_CATEGORY_2 = "image";
+
+    private static final String ISR_CATEGORY_3 = "video";
+
+    private static final String ISR_COMMENTS_1 = "height above reference surface in something else";
+
+    private static final String ISR_COMMENTS_2 =
+            "optical reflective surface mgmp image description";
+
+    private static final String ISR_COMMENTS_3 = "monochromatic value of pixel";
+
+    private static final String ISR_COMMENTS_4 = "horizontal video, reflective surface";
+
+    private static final String ORGANISATION_NAME = "Codice Foundation";
+
+    private static final String METACARD_ID = "a metacard";
+
+    private static final MetacardType MGMP_METACARD_TYPE = getMgmpMetacardType();
+
+    private static final Double ALTITUDE_1 = 305.0;
+
+    private static final Double ALTITUDE_2 = 312.0;
+
+    private static final String OFFICIAL_SENSITIVE = "officialSensitive";
+
+    private static final String NATO_SECRET = "NATOSecret";
+
+    private static final String OFFICIAL = "official";
+
+    private static final String UNCLASSIFIED = "unclassified";
+
+    @Test
+    public void testInputTransformerWithFullMgmpMetacard() throws Exception {
+        InputStream fileInput =
+                TestMgmpTransformer.class.getResourceAsStream("/full_data_mgmp.xml");
+        MgmpTransformer mgmpTransformer = new MgmpTransformer(MGMP_METACARD_TYPE);
+        Metacard metacard = mgmpTransformer.transform(fileInput);
+
+        assertThat(metacard.getTitle(), is("Surrey TLM"));
+        assertThat(metacard.getId(), is("4d40336f-70c2-488c-835a-1cff24b90620"));
+        assertThat(convertDate(metacard.getModifiedDate()), is("2014-10-31 12:00:00 UTC"));
+        assertThat(convertDate(metacard.getExpirationDate()), is("2014-12-11 00:00:00 UTC"));
+        assertThat(convertDate(metacard.getCreatedDate()), is("2000-12-01 00:00:00 UTC"));
+        assertThat(convertDate(metacard.getEffectiveDate()), is("1998-01-01 07:00:00 UTC"));
+        Date createdDate = (Date) metacard.getAttribute(Core.METACARD_CREATED)
+                .getValue();
+        Date modifiedDate = (Date) metacard.getAttribute(Core.METACARD_MODIFIED)
+                .getValue();
+        assertThat(convertDate(createdDate), is("2015-07-24 00:00:00 UTC"));
+        assertThat(convertDate(modifiedDate), is("2015-07-24 00:00:00 UTC"));
+
+        assertThat(metacard.getAttribute(Core.LANGUAGE)
+                .getValues(), hasItems("ger", "eng"));
```

Comment:
Are these defined by the specification to be lower case? 

I know your input data and test are geared to work together, but insofar as unit tests often serve as additional documentation to developers, it might be good to be clear in this case if case is fixed or to test in a case-insensitive manner.